docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3.7'
+services:
+  app:
+    container_name: od-app-container
+    image: od-app:latest
```

Comment:
open-dollar-app

Reply:
I agree on this one. So what I can do is to update `image: open-dollar-app:latest` to pull the latest version when `docker compose up` is executed. If there is another name that we think is better, we can update it before publishing the image to the registry.